config.json

```diff
@@ -38,6 +38,25 @@
                 "WINDOW_SIZE": 1440
             }
         },
+         {
+            "ID": "GOOD Test",
+            "NAME": "GOOD CARE Testnet",
+            "TOKEN": "GOOD",
+            "RPC": "https://subnets.avax.network/goodtest/testnet/rpc",
+            "CHAINID": 48169,
+            "EXPLORER": "https://subnets-test.avax.network/goodtest",
+            "IMAGE": "https://bit.ly/goodcarelogo",
+            "MAX_PRIORITY_FEE": "25000000",
+            "MAX_FEE": "250000000",
+            "DRIP_AMOUNT": 1000,
+            "DECIMALS": 18,
+            "RECALIBRATE": 30,
+            "COUPON_REQUIRED": false,
+            "MAINNET_BALANCE_CHECK_ENABLED": false,
+            "RATELIMIT": {
+                "MAX_LIMIT": 1,
+                "WINDOW_SIZE": 1440
+            },
```

Comment:
It looks like the configuration is not properly closed. Can you add a parenthesis and remove the comma?